meteor/lib/Settings.ts

```diff
@@ -82,10 +82,11 @@ const DEFAULT_SETTINGS = Object.freeze<ISettings>({
 	maximumDataAge: 1000 * 60 * 60 * 24 * 100, // 100 days
 	enableNTPTimeChecker: null,
 	defaultShelfDisplayOptions: 'buckets,layout,shelfLayout,inspector',
-	enableKeyboardPreview: false,
+	enableKeyboardPreview: true,
```

Comment:
Reintroducing conflicts. Please don't do this.

Reply:
Please tell me what is the point of having a place to specify the values that is specific to your system if you are not allowed to use them? This is TV2s repo, why must there be NRK values in this file?

Also, there will be some files we can't avoid conflicts in. For me, this is an obvious one

Reply:
We have a feature that is made to prevent conflicts in this file. core-settings.json is the place to specify values specific to your system.